pkgs/build-support/fetch9front/default.nix

```diff
@@ -0,0 +1,37 @@
+{ fetchgit, fetchzip, lib }:
+
+lib.makeOverridable (
+  { owner
+  , repo
+  , rev
+  , protocol ? "git"
```

Comment:
Can this be anything other than git?

Reply:
In theory yes, but the other protocols require git extensions to work. I will remove this argument for the time being.